search_gov_crawler/requirements.txt

```diff
@@ -10,3 +10,5 @@ scrapy-playwright==0.0.41
 scrapyd==1.5.0
 scrapyd-client==2.0.0
 scrapydweb @ git+https://github.com/GSA/searchgov-scrapydweb
+spidermon>=1.22.0
+premailer>=3.10.0
```

Comment:
can you change these to `==`?  As is I would worry that, for example, a spidermon upgrade to v2 with breaking changes would get installed during a deploy and break things.

Reply:
will do!

Reply:
why do we need premailer referenced in our project?  i didn't see any other reference to it.

Reply:
Premailer is used by spidermon but they do not import it so we have to

Reply:
i didnt try it but you should be able to have it installed by changing it to 
from `spidermon` to `"spidermon[monitoring]"`

https://spidermon.readthedocs.io/en/latest/installation.html

Looks like it will install some extra libraries including premailer. https://github.com/scrapinghub/spidermon/blob/de6921541f38613f62384efc682ed2a0282b08fa/setup.py#L20